synapse/storage/util/id_generators.py

```diff
@@ -433,6 +433,9 @@ def __init__(
             # position with the current minimum.
             self._current_positions[self._instance_name] = self._persisted_upto_position
 
+        table_string = "\n".join(", ".join(row) for row in tables)
+        logger.debug(f"{table_string}")
```

Comment:
I'm not sure if this is helpful logging or not -- it seems to just be spitting back out what was input?

Maybe something like:

```
logger.info("Initialising multi-writer generator for %s: %i (%s)", stream_name, self.persisted_upto_position, self._current_positions)
```

I'm unsure this would be helpful or not. @erikjohnston any thoughts?